components/tyk-mdcb/values.yaml

```diff
@@ -184,7 +184,17 @@ mdcb:
 
   # If enabled, aggregate data will be generated per minute.
   storeAnalyticsPerMinute: false   
-
+ 
+  # defines the SSL/TLS settings for the http server where the healthcheck is exposed
+  httpServerOptions:
+    # if enabled then the endpoints will be served over https
+    useSSL: false
+    # certificateKeyFile: /path-to-cert-keyfile
+    # certificateCertFile: /path-to-certfile
+    
+    # For TLS 1.0 use 769, for TLS 1.1 use 770, for TLS 1.2 use 771, for TLS 1.3 use 772
+    # minVersion: 771
```

Comment:
we do support more config options for SSL, am wondering why they are not here. Do we support them? Eg: `TYK_MDCB_HTTPSERVEROPTIONS_CIPHERS` etc

Reply:
It could be configured by using `extraEnv` field in which user can provide environment variables they want to set directly.
But if you think we should support by adding new field for it in values.yaml, you can raise a ticket for it.